iotdb/src/main/java/org/apache/iotdb/db/engine/memtable/AbstractMemTable.java

```diff
@@ -94,4 +101,58 @@ public TimeValuePairSorter query(String deviceId, String measurement, TSDataType
     return memTableMap.get(deviceId).get(measurement);
   }
 
+  @Override
+  public void delele(String deviceId, String measurementId, long timestamp) {
+    Map<String, IWritableMemChunk> deviceMap = memTableMap.get(deviceId);
+    if (deviceMap != null) {
+      IWritableMemChunk chunk = deviceMap.get(measurementId);
+      IWritableMemChunk newChunk = filterChunk(chunk, timestamp);
+      if (newChunk != null) {
+        deviceMap.put(measurementId, newChunk);
+      }
```

Comment:
so, how to guarantee the concurrency control.

Reply:
The lock of a FileNode must be acquired at the beginning of deletion.